### DISCUSSION TITLE
I get a Photo problem while trying to upload

### DISCUSSION THREAD

**Discussion by PavelS05** [community/community#20337] [GitHub Education]
I'm trying to upload a photo for Student developer pack but when I click 'Process my application' it says this:

```
Your application cannot be reviewed untill you fix one of the following:
- Your image should include a date within the range of 2018 through 2026
- Your image should include the name of your school
```
even if my image includes name of the school and a valid date 2018-2026.

Photo I'm trying to upload contains my face and school ID(they call it 'Carnet de Elev' and I'm using it to get in)

![image](https://user-images.githubusercontent.com/98814610/178275673-ee1f0ed8-a929-44be-94e7-4d6b0076e0e6.png)

**Comment by D7EAD** [comment]
Hi!

I'd sugget contacting Github Education's Support Engineers directly to resolve this issue. You can do so [here](https://support.github.com/contact/education).

**Comment by FahdSeddik** [comment]
Hey!
I suggest you follow the instructions provided in this [link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack) . You should find a section where they provide examples for the picture. Hope this helps!
